samples/bidirectional-chat/csharp/Function.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.WebJobs.Extensions.SignalRService;
+using Microsoft.AspNetCore.SignalR;
+
+namespace FunctionApp
+{
+    public class SimpleChat : ServerlessHub
+    {
+        private const string Hub = nameof(SimpleChat);
+        private const string NewMessageTarget = "newMessage";
+        private const string NewConnectionTarget = "newConnection";
+
+        [FunctionName("negotiate")]
+        public static SignalRConnectionInfo GetSignalRInfo(
```

Comment:
Can this be moved into ServerlessHub?

Reply:
That's good. But it needs some investigation. I will put it into another RP.